packages/ui/src/ui/pages/query-tracker/QueryMetaTable/index.tsx

```diff
@@ -21,13 +22,16 @@ const valueClassName = b('value');
 type QueryMetaItem = {
     title: string;
     value: string | number | boolean;
+    showCopy: boolean;
 };
 
 const exceptFields: (keyof QueryItem | 'access_control_object')[] = [
     'query',
     'access_control_object',
 ];
 
+const FIELDS_WITH_COPY_BUTTON = ['id'];
```

Comment:
Is it planned that there will be more such fields? Now it would be easier to use string.